tox.ini

```diff
@@ -1,18 +1,20 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,pypy3
+   py27,py34,py35,py36,pypy,pypy3,coverage
```

Comment:
The alignment change to a 3-space indent looks strange here.

Reply:
I don't know what happened there. Thanks!